lib/recording/RecordingEntrySession.js

```diff
@@ -27,7 +27,7 @@ const recordingMaxDurationInMs = config.get('recording').recordingMaxDurationInH
 const unknownEntry = "UNKNOWN";
 
 class RecordingEntrySession {
-    constructor(entryObj, mp4FilesQueue = null) {
+    constructor(entryObj, entryServerNodeId, mp4FilesQueue = null) {
```

Comment:
I saw that RestoreRecordingScript.js and Recording-spec.js have references to this constructor and don't seem to be updated. I don't know if those files are still relevant.